<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserdataDialog</class>
 <widget class="QDialog" name="UserdataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Little Navmap - Userpoint</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="17" column="0" colspan="4">
      <widget class="QLabel" name="labelUserdataTemp">
       <property name="text">
        <string/>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QLabel" name="labelUserdataVisible">
       <property name="text">
        <string>&amp;Visible from:</string>
       </property>
       <property name="buddy">
        <cstring>spinBoxUserdataVisible</cstring>
       </property>
      </widget>
     </item>
     <item row="15" column="2">
      <widget class="QLabel" name="labelUserdataLastChangeDateTime">
       <property name="text">
        <string/>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QSpinBox" name="spinBoxUserdataAltitude">
       <property name="toolTip">
        <string>Elevation or altitude of this waypoint</string>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>true</bool>
       </property>
       <property name="suffix">
        <string> %alt%</string>
       </property>
       <property name="minimum">
        <number>-2000</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="lineEditUserdataName"/>
     </item>
     <item row="2" column="0" colspan="4">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="15" column="0" colspan="2">
      <widget class="QLabel" name="labelUserdataLastChange">
       <property name="text">
        <string>Last Change:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QLabel" name="labelUserdataCoordStatus">
       <property name="text">
        <string/>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QCheckBox" name="checkBoxUserdataTags">
       <property name="toolTip">
        <string>Check to change tags for all selected waypoints</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="16" column="2">
      <widget class="QLabel" name="labelUserdataFilepath">
       <property name="text">
        <string/>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QCheckBox" name="checkBoxUserdataName">
       <property name="toolTip">
        <string>Check to change name for all selected waypoints</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEditUserdataRegion">
       <property name="toolTip">
        <string>Two letter ICAO region code.
Required for export to X-Plane &quot;user_fix.dat&quot;.</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QCheckBox" name="checkBoxUserdataVisible">
       <property name="toolTip">
        <string>Check to change visibility for all selected waypoints</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QLabel" name="labelUserdataLatLon">
       <property name="text">
        <string>&amp;Latitude and Longitude:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditUserdataLatLon</cstring>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QLabel" name="labelUserdataAltitude">
       <property name="text">
        <string>&amp;Elevation or altitude:</string>
       </property>
       <property name="buddy">
        <cstring>spinBoxUserdataAltitude</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QComboBox" name="comboBoxUserdataType">
       <property name="toolTip">
        <string>Type which will determine the used icon</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="maxVisibleItems">
        <number>20</number>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLineEdit" name="lineEditUserdataTags">
       <property name="toolTip">
        <string>List of tags. Simple textfield for various purposes.

Tags are also used for X-Plane export and import to and from &quot;user_fix.dat&quot;.
The field &quot;Tags&quot; is used for the ID of the airport terminal area and the waypoint type.
Click &quot;Help&quot; for more information.</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="labelUserdataType">
       <property name="text">
        <string>&amp;Type:</string>
       </property>
       <property name="buddy">
        <cstring>comboBoxUserdataType</cstring>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QCheckBox" name="checkBoxUserdataAltitude">
       <property name="toolTip">
        <string>Check to change elevation for all selected waypoints</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QCheckBox" name="checkBoxUserdataType">
       <property name="toolTip">
        <string>Check to change type for all selected waypoints</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QLabel" name="labelUserdataTags">
       <property name="text">
        <string>&amp;Tags:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditUserdataTags</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="lineEditUserdataIdent">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ident code of the waypoint. &lt;/p&gt;&lt;p&gt;An up to five character ident is required for export to other formats for X-Plane &amp;quot;user_fix.dat&amp;quot; or Garmin &amp;quot;user.wpt&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="16" column="0" colspan="2">
      <widget class="QLabel" name="labelUserdataFile">
       <property name="text">
        <string>Imported from file:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="labelUserdataIdent">
       <property name="text">
        <string>&amp;Ident:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditUserdataIdent</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QSpinBox" name="spinBoxUserdataVisible">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Define the maximum view distance  where this waypoint is shown on the map.&lt;/p&gt;&lt;p&gt;The view distance is shown in the status bar.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>true</bool>
       </property>
       <property name="suffix">
        <string> %dist%</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>3000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item row="14" column="0" colspan="4">
      <widget class="Line" name="lineUserdataMeta">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="labelUserdataRegion">
       <property name="text">
        <string>&amp;Region:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditUserdataRegion</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QCheckBox" name="checkBoxUserdataRegion">
       <property name="toolTip">
        <string>Check to change region for all selected waypoints</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QLabel" name="labelUserdataDescription">
       <property name="text">
        <string>&amp;Remarks:</string>
       </property>
       <property name="buddy">
        <cstring>plainTextEditUserdataDescription</cstring>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="4">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QLineEdit" name="lineEditUserdataLatLon">
       <property name="toolTip">
        <string>Recognized coordinate formats:
- Degrees, minutes and seconds : N49° 26' 41.57&quot; E9° 12' 5.49&quot; or 49° 26' 41.57&quot; N 9° 12' 5.49&quot; E
- Degrees and minutes: N54* 16.82' W008* 35.95', N 52 33.58 E 13 17.26 or 49° 26.69' N 9° 12.09' E
- Degrees only: 49.4449° N 9.2015° E or N 49.4449° E 9.2015°
- Degrees with sign: 49.4449 9.2015. Negative is west or south. Order for latitude and longitude depends on option settings.
Degree, minute and second signs can be omitted for above formats.
Latitude has to be first and N/S and E/W designators are required.
A space is only needed to separate latitude and longitude or degrees/minutes/seconds if no signs used.
Other formats:
- Degrees and minutes: N44124W122451, N14544W017479 or S31240E136502
- Degrees only 46N078W
- Degrees and minutes 4620N07805W
- Degrees, minutes and seconds 481200N0112842E (SkyVector)
- Degrees and minutes in pair &quot;N6500 W08000&quot; or &quot;N6500/W08000&quot;
- NAT type 5020N
Formats above have to be given exactly as shown in the examples.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QCheckBox" name="checkBoxUserdataIdent">
       <property name="toolTip">
        <string>Check to change ident for all selected waypoints</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QCheckBox" name="checkBoxUserdataDescription">
       <property name="toolTip">
        <string>Check to change remarks for all selected waypoints</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="labelUserdataName">
       <property name="text">
        <string>&amp;Name:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditUserdataName</cstring>
       </property>
      </widget>
     </item>
     <item row="13" column="2">
      <widget class="QLabel" name="labelUserdataMagVar">
       <property name="text">
        <string/>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPlainTextEdit" name="plainTextEditUserdataDescription">
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxUserdataTemp">
     <property name="text">
      <string>&amp;Temporary userpoint. Delete on next startup.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBoxUserdata">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditUserdataIdent</tabstop>
  <tabstop>checkBoxUserdataIdent</tabstop>
  <tabstop>lineEditUserdataRegion</tabstop>
  <tabstop>checkBoxUserdataRegion</tabstop>
  <tabstop>comboBoxUserdataType</tabstop>
  <tabstop>checkBoxUserdataType</tabstop>
  <tabstop>lineEditUserdataName</tabstop>
  <tabstop>checkBoxUserdataName</tabstop>
  <tabstop>plainTextEditUserdataDescription</tabstop>
  <tabstop>checkBoxUserdataDescription</tabstop>
  <tabstop>lineEditUserdataTags</tabstop>
  <tabstop>checkBoxUserdataTags</tabstop>
  <tabstop>spinBoxUserdataVisible</tabstop>
  <tabstop>checkBoxUserdataVisible</tabstop>
  <tabstop>spinBoxUserdataAltitude</tabstop>
  <tabstop>checkBoxUserdataAltitude</tabstop>
  <tabstop>lineEditUserdataLatLon</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
